snapshot-labs/snapshot
# Show meaningful error when votes download is failing on the demo website

### Issues

Fixes #4069 

Votes report download is not working on the demo website. It will not work, as votes report are available only on the main network on sidekick side.

We should show a meaningful error message, and avoid triggering the download request, on the demo website.

### Changes 

1. Add a filter to not trigger the download request on demo environment
2. Show a `unsupported network` error message when trying to download a votes report on demo env

### How to test

1. Change your local instance env to demo
2. Open the app with yarn dev
3. To to a proposal
4. Click on the votes report download button
5. It should show a message saying that the network is not supported
6. If you open the dev network tab, you should also not see a request to sidekick server

This is not testable on the vercel/fleek preview, as they're not on the correct 'demo' env to show the correct message

### Self-review checklist

### Additional notes or considerations
*_(Include any other relevant information or context that may be helpful for the reviewer)_*
